engine/src/main/java/org/terasology/config/facade/BindsConfiguration.java

```diff
@@ -10,24 +10,54 @@
 import java.util.List;
 
 /**
- * Facade for {@link Config#getBinds()}
+ * Facade for {@link Config#getBinds()}.
+ * The binds configuration holds the mapping from bind uris.
+ * The {@link SimpleUri} for a binding contains the module from the binding and the id from the binding annotation, 
+ * e.g. {@link RegisterBindButton.
  */
 public interface BindsConfiguration {
 
+    /**
+     * Returns true if the input is bound to a bind uri.
+     */
     boolean isBound(Input newInput);
 
+    /**
+     * Sets all bindings to the mappings from the given config.
+     */
     void setBinds(BindsConfig other);
 
+    /**
+     * Returns a list of all input, bound to the given bind uri.
+     */
     List<Input> getBinds(SimpleUri uri);
 
+    /**
+     * Returns the internal binds config.
```

Comment:
I take it binds and bindings are used interchangeably in this part of the codebase. I checked on dictionary.com and the term "bind" can be a noun. I just feel it's a bit ambiguous and here I'd recommend the term "bindings" instead, leaving "bind" to be used only as a verb. But I don't want to increase too much the scope of this PR - sorry for the diversion into the dictionary.